docs/getting-started/security-ui.asciidoc

```diff
@@ -22,204 +23,69 @@ image::images/search-bar.png[]
+* <<users-page, *Users*>>: Access a comprehensive overview of user data to help you understand authentication and user behavior within your environment.
 
 [role="screenshot"]
 image::images/users/users-page.png[Shows the Users page]
 
+*Get started page*: View a quick shortcut to add security integrations, which you can use to monitor your hosts and ingest data. Other {security-app} pages that show event data (for example, the Hosts and Network pages) display the get started prompt until event data has been ingested.
 
-[float]
-[[timelines-page]]
-=== Timelines page
-
-Use the Timelines page to investigate alerts and complex threats, such as lateral movement of malware across hosts in your network. Timelines are responsive and allow you to share your findings among other team members. Refer to <<timelines-ui, Investigate events in Timeline>> for information about getting started with Timelines.
-
-TIP: Select the collapsable *Timeline* button at the bottom of the {security-app} to start an investigation.
-
-[role="screenshot"]
-image::images/timeline-ui.png[Shows the Timeline page]
-
-[float]
-[[cases-ui]]
-=== Cases page
-
-The Cases page is used to open and track security issues directly in the {security-app}. Refer to <<cases-overview, Cases>> for more information.
-
-[role="screenshot"]
-image::cases/images/cases-home-page.png[Shows the Cases page]
-
-[float]
-[[endpoints-page]]
-=== Endpoints page
-
-The Endpoints page allows you to view and manage hosts running {endpoint-cloud-sec}. Refer to <<admin-page-ov, Endpoints>> for more information.
-
-[role="screenshot"]
-image::management/admin/images/endpoints-pg.png[Shows the Endpoints page]
-
-[float]
-[[policies-page]]
-=== Policies page
-
-The Policies page lists all of the integration policies configured for {endpoint-cloud-sec}.  Refer to <<policies-page-ov, Policies>> for more information.
-
-[role="screenshot"]
-image::management/admin/images/policy-list.png[Policies page]
-
-[float]
-[[trusted-apps-page]]
-=== Trusted applications page
-
-The Trusted applications page allows you to add Windows, macOS, and Linux applications that should be trusted. Refer to <<trusted-apps-ov, Trusted applications>> for more information.
-
-[role="screenshot"]
-image::management/admin/images/trusted-apps-list.png[Shows the Trusted applications page]
-
-[float]
-[[event-filters-page]]
-=== Event filters page
-
-The Event filters page allows you to filter endpoint events that you do not need or want stored in {es}. Refer to <<event-filters, Event filters>> for more information.
-
-[role="screenshot"]
-image::management/admin/images/event-filters-list.png[Shows the Event filters page]
+image::images/getting-started-pg.png[Shows the Get started page]
 
-[float]
-[[host-isolation-exceptions-page]]
-=== Host isolation exceptions page
+*Manage*: Expand this section to manage these frequently used tools in the {security-app}:
```

Comment:
```suggestion
*Manage*: Expand this section to manage the {security-app}:
```
I could be off here, but calling all these features "tools" struck me as a bit unusual.

Reply:
I still think we should keep that part of the sentence. The latter almost sounds as though we're directing users towards the application settings. How about: 

```suggestion
*Manage*: Expand this section to access and manage additional security features:
```